### PR TITLE
feat(ui): input labels

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/react-native-ui-kitten/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/react-native-ui-kitten/blob/master/DEV_DOCS.md) guide.

 #### Short description of what this resolves:
* ability to add label implemented
* ability to add caption/caption-image implemented
* "size" issues fixed

### CODE REVIEW

File: src/framework/ui/common/mapping.json [hunk 1]
```diff
@@ -685,7 +685,7 @@
       "appearances": {
         "filled": {
           "mapping": {
-            "buttonBorderRadius": 6
+            "buttonBorderRadius": 8
```
Comment (artyorsh) [inline src/framework/ui/common/mapping.json]: ?

File: src/framework/ui/input/input.component.tsx [hunk 2]
```diff
@@ -82,52 +119,108 @@ export class Input extends React.Component<Props> {
...
+        },
+        icon: {
+          width: captionIconWidth,
+          height: captionIconHeight,
+          tintColor: captionIconTintColor,
+          marginRight: captionIconMarginRight,
+        },
+        container: {
+          marginTop: captionMarginTop,
+          ...styles.caption,
```
Comment (artyorsh) [inline src/framework/ui/input/input.component.tsx]: container i think

File: src/framework/ui/input/input.component.tsx [hunk 3]
```diff
@@ -82,52 +119,108 @@ export class Input extends React.Component<Props> {
...
+        marginBottom: labelMarginBottom,
+        fontWeight: labelFontWeight,
+        ...styles.label,
+      },
+      caption: {
+        text: {
+          color: captionTextColor,
+          fontSize: captionTextFontSize,
+          fontWeight: captionTextFontWeight,
+          lineHeight: captionTextLineHeight,
```
Comment (artyorsh) [inline src/framework/ui/input/input.component.tsx]: ...styles.caption?

File: src/framework/ui/input/input.component.tsx [hunk 4]
```diff
@@ -82,52 +119,108 @@ export class Input extends React.Component<Props> {
...
       </View>
     );
   }
 }
 
 const styles = StyleSheet.create({
+  wrapper: {
+    flexGrow: 1,
+  },
   container: {
```
Comment (artyorsh) [inline src/framework/ui/input/input.component.tsx]: rename this to `inputContainer` and `wrapper` to `container`

File: src/framework/ui/common/mapping.json (hunk 1 above)
Comment (32penkin) [reply src/framework/ui/common/mapping.json]: It's a correct value

File: src/framework/ui/input/input.component.tsx (hunk 2 above)
Comment (32penkin) [reply src/framework/ui/input/input.component.tsx]: Refactored

File: src/framework/ui/input/input.component.tsx (hunk 3 above)
Comment (32penkin) [reply src/framework/ui/input/input.component.tsx]: Refactored

File: src/framework/ui/input/input.component.tsx (hunk 4 above)
Comment (32penkin) [reply src/framework/ui/input/input.component.tsx]: Done.

**Pull request merged**